### PR TITLE
More dependencies listed and sample nginx configuration added in the installation doc.

### PR DESCRIPTION
**Pull request by umpc**

### CODE REVIEW

**Comment by cydrobolt** [comment]
This looks good, but perhaps we want to put SSL in an optional section for those who don't want it or have certificates?

File: docs/user-guide/installation.md [hunk 1]
```diff
@@ -11,23 +11,22 @@ you may be interested in looking at a [legacy 1.x release](https://github.com/cy
 
  - Apache, nginx, IIS, or lighttpd (Apache preferred)
  - PHP >= 5.5.9
- - MariaDB >= 10.0 or MySQL >= 5.5, SQLite alternatively
+ - MariaDB or MySQL >= 5.5, SQLite alternatively
  - Composer (optional)
  - PHP requirements:
     - OpenSSL PHP Extension
     - PDO PHP Extension
+    - PDO MySQL Driver (php-mysql on Debian & Ubuntu, php5x-pdo_mysql on FreeBSD)
```
Comment (cydrobolt) [inline docs/user-guide/installation.md]: I think it is actually `php5-mysql` for Debian and Ubuntu

**Comment by umpc** [comment]
I left it in the standard example because there's no privacy for login credentials without TLS, but that's just a personal preference. Though I don't suppose too many people would be sniffed for URL shortening credentials.

I put the package that was on Debian's package repo site. It might need the 5 version number, but I didn't see it on the package repo site in the package name last night.

**Comment by umpc** [comment]
I misread. You're right, it is php5-mysql.

**Comment by cydrobolt** [comment]
We could perhaps comment out the SSL section, so they those who want it can use it. Otherwise, we would also have to help out all the people who don't understand how to generate SSL certs and the like, which isn't in the scope of the guide

**Comment by cydrobolt** [comment]
Thanks @umpc !

**Pull request merged**